### PR TITLE
Update test reference to 1.7.0-beta.1

### PR DESCRIPTION
**Pull request by ericvn**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
circuit breaker failed but the test update wasn't in yet when this started.

**Comment by ericvn** [comment]
Possible breakage in `TestDocs/ops/diagnostic-tools/istioctl-analyze/test.sh`. More investigation soon.

File: content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ spec:
 status:
   validationMessages:
   - code: IST0101
-    documentation_url: https://istio.io/docs/reference/config/analysis/IST0101?ref=status-controller
+    documentation_url: https://istio.io/docs/reference/config/analysis/ist0101?ref=status-controller
```
Comment (frankbu) [inline content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md:101]:
```suggestion
    documentation_url: https://istio.io/docs/reference/config/analysis/ist0101/?ref=status-controller
```

Comment (ericvn) [reply content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md]: Thanks @frankbu . Hard to find the missing `/` in the output.

**Pull request merged**